core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1714,6 +1715,19 @@ public ServerBuilder setHeaders(
      * @see RequestContext#id()
      */
     public ServerBuilder requestIdGenerator(Supplier<? extends RequestId> requestIdGenerator) {
+        final Supplier<? extends RequestId> requestIdSupplier = requireNonNull(requestIdGenerator);
+        this.requestIdGenerator = routingContext -> requestIdSupplier.get();
+        return this;
+    }
+
+    /**
+     * Generate {@link RequestId} from {@link RoutingContext}.
+     * By default, a {@link RequestId} is generated from a random 64-bit integer.
```

Comment:
Should we update the Javadoc to describe the behavior of this method?
In fact, this method does not generate `RequestId` but takes a request ID generator.

Reply:
Sure! I modify it.